### PR TITLE
Add some words

### PR DESCRIPTION
**Pull request by Drulac**
Add some words

### CODE REVIEW

File: data/words/personals_pronouns.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+[{
+	"word": "i",
```
Comment (hugovk) [inline data/words/personals_pronouns.json]: Please capitalise this as `I`

File: data/words/personals_pronouns.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	"people": "f",
+	"gender": "",
+	"number": "s"
```
Comment (hugovk) [inline data/words/personals_pronouns.json]: What do the `people` and `number` values represent?

File: data/words/state-verbs.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+[
+	{
+		"word": "been",
+		"verb-type": "state-verb",
+		"people": "",
+		"number": ""
```
Comment (hugovk) [inline data/words/state-verbs.json]: If `people` and `number` aren't used here, can they be removed?

File: data/words/verbs_with_conjugaisons.json [hunk 4]
```diff
@@ -0,0 +1 @@
...mperfecto":["camped","camped","camped","camped","camped","camped"],"futur":["camp","camp","camp","camp","camp","camp"],"Futuro anterior":["camped","camped","camped","camped","camped","camped"]},"subjuntive":{"Presente":["camp","camp","camp","camp","camp","camp"],"Perfecto":["camped","camped","camped","camped","camped","camped"]},"condicional":{"Condicional":["camp","camp","camp","camp","camp","camp"],"Condicional perfecto":["camped","camped","camped","camped","camped","camped"]},"imperative":["camp","camp","camp"]},{"verb":"care","infinitive":["care"],"participio":["cared"],"gerundif":["caring"],"indicative":{"present":["care","care","cares","care","care","care"],"perfecto":["cared","cared","cared","cared","cared","cared"],"imperfect":["cared","cared","cared","cared","cared","cared"],"pluscuamperfecto":["cared","cared","cared","cared","cared","cared"],"futur":["care","care","care","care","care","care"],"Futuro anterior":["cared","cared","cared","cared","cared","cared"]},"subjuntive":{"Presente":["care","care","care","care","care","care"],"Perfecto":["cared","cared","cared","cared","cared","cared"]},"condicional":{"Condicional":["care","care","care","care","care","care"],"Condiciona
```
Comment (hugovk) [inline data/words/verbs_with_conjugaisons.json]: Can the keys like `condicional`, `subjuntive`, `pluscuamperfecto` etc., be translated to English?

`undefineded` should be `undefined`.

**Comment by Drulac** [comment]
For the Travis error, it's normal, there is a lot of verb who have the infinitv a lot of time ;-)

**Comment by Drulac** [comment]
up ;-)

**Comment by hugovk** [comment]
This was merged as part of https://github.com/dariusk/corpora/pull/306. Thank you!

**Pull request closed without merging**